presto-main/src/test/java/com/facebook/presto/sql/planner/TestTypeValidator.java

```diff
@@ -68,7 +70,11 @@
 @Test(singleThreaded = true)
 public class TestTypeValidator
 {
-    private static final TableHandle TEST_TABLE_HANDLE = new TableHandle(new ConnectorId("test"), new TestingTableHandle());
+    private static final TableHandle TEST_TABLE_HANDLE = new TableHandle(
+            new ConnectorId("test"),
+            new TestingTableHandle(),
+            TestingTransactionHandle.create(),
+            Optional.of(TestingHandle.INSTANCE));
```

Comment:
should this also be `Optional.empty()` ? I note https://github.com/prestosql/presto/pull/363/commits/47ce7e62491691eb91126c68db09f5eca1e080ff is also not consistent on whether to use `Optional.empty` or `Optional.of(TestingHandle)`. We can discuss in person.